.travis.yml

```diff
@@ -20,6 +22,9 @@ before_script:
   - sudo dpkg -i ./vroom_0.12.0-1_all.deb
   - git clone -b ${MAKTABA_VERSION} https://github.com/google/vim-maktaba.git ../maktaba/
   - git clone https://github.com/google/vim-glaive.git ../glaive/
+  - wget https://github.com/kkinnear/zprint/releases/download/${ZPRINT_VERSION}/zprintl-${ZPRINT_VERSION}
+  - sudo mv zprintl-${ZPRINT_VERSION} /usr/local/bin/zprint
+  - sudo chmod +x /usr/local/bin/zprint
```

Comment:
I think the GNU `install` command is supposed to move and chmod files in one go. Can you check if that will work here?

Reply:
I had considered that, but GNU `install` copies rather than moves and that seemed messy. I don't have strong feelings about it—just let me know what you'd like.

Reply:
Done